### DISCUSSION TITLE
Pin to background as a widget

### DISCUSSION THREAD

**Discussion by derrickmg** [WindowTop/WindowTop-App#321] [Ideas]
This is especially useful if you have more than one monitor, but equally can be used with virtual desktops.
Now the beta has cropping It would be cool to pin a Spotify "widget" next to a WhatsApp/ weather/stock widget etcetera.

So instead of pinning to the top and above other windows these would be behind and fixed in place.

**Comment by gileli121** [comment]
@derrickmg 
It should be possible and relatively easy to implement 

But I have a hard time understanding what kind of advantage you get by forcing a window to be on the desktop. 

What is the difference between having it a regular cropped window without always on top and having it bound to the desktop?

Maybe it may come in handy if you click on the "Show desktop" button
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/dbf78e35-7890-4e37-803f-9538d369fbd9)
and in such a case the "widget" window will keep showing?

Not sure... I am trying to see what benefits it gives.
